src/main/fc/rc.c

```diff
@@ -94,7 +94,7 @@ enum {
 #define RC_SMOOTHING_RX_RATE_CHANGE_PERCENT     20    // Look for samples varying this much from the current detected frame rate to initiate retraining
 #define RC_SMOOTHING_RX_RATE_MIN_US             1000  // 1ms
 #define RC_SMOOTHING_RX_RATE_MAX_US             50000 // 50ms or 20hz
-#define RC_SMOOTHING_INTERPOLATED_FEEDFORWARD_DERIVATIVE_PT1_HZ 100 // The value to use for "auto" when interpolated feedforward is enabled
+#define RC_SMOOTHING_INTERPOLATED_FEEDFORWARD_INITIAL_HZ 50 // The initial value for "auto" when interpolated feedforward is enabled
```

Comment:
Re-named the setting and set it to 50hz.
If auto smoothing is zero (default), derivative will get 50hz smoothing initially whether PT1 or biquad.
If auto smoothing is 50 (previous max) derivative will be 25Hz.
Those are not hazardous defaults.
In any case will now be updated once RC rate is known.